### PR TITLE
Add RID for Windows Arm64

### PR DESCRIPTION
**Pull request by gkhanna79**
@ericstj @weshaggard PTAL - this adds RID for Windows ARM64

Fixes https://github.com/dotnet/corefx/issues/6889

CC @haripulapaka

### CODE REVIEW

File: pkg/Microsoft.NETCore.Platforms/runtime.json [hunk 1]
```diff
@@ -129,6 +131,9 @@
         "win10-arm-aot": {
             "#import": [ "win10-aot", "win10-arm", "win81-arm-aot" ]
         },
+        "win10-arm64-aot": {
+            "#import": [ "win10-aot" ]
```
Comment (weshaggard) [inline pkg/Microsoft.NETCore.Platforms/runtime.json]: Shouldn't this also contain win10-arm64 in the list?

Comment (gkhanna79) [inline pkg/Microsoft.NETCore.Platforms/runtime.json]: You are right - I removed an extra fallback RID.

**Comment by gkhanna79** [comment]
@weshaggard @ericstj Thoughts on the change?

**Comment by weshaggard** [comment]
LGTM assuming we don't want these to import win81 as well.

**Comment by gkhanna79** [comment]
> we don't want these to import win81 as well.

Correct - there is no precedence there as this is the minimal platform.

**Pull request merged**